src/main/java/upb/dice/twitter/TweetExtraction.java

```diff
@@ -0,0 +1,56 @@
+package upb.dice.twitter;
+
+import twitter4j.*;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+import static twitter4j.TwitterFactory.getSingleton;
+
+/**
+ * Extract tweets based on the keywords or location(provided as latitude, longitude and the radius of coverage)
+ */
+public class TweetExtraction {
+    private StoreMaxID storeMaxID = new StoreMaxID();
+    private Twitter twitter = getSingleton();
```

Comment:
Please use `TwitterFactory.getSingleton();` instead. Otherwise it is slightly confusing where the method comes from.